### DISCUSSION TITLE
[#991922] Adjust WASD keybindings to be inclusive for non-QWERTY keybaords

### DISCUSSION THREAD

**Discussion by Landy005** [Mojang/minecraft-editor#45] [Feedback]
Perhaps this is intended and this is more like a feature request. But I thought to let you know just in case this is a bug;

I have an azerty keyboard and even though I set and changed my Minecraft preview keyboard to azerty it still forces the qwerty controls to control the editor. The controls are very intuitive as they are similar to MCedit/Amulet but lack customisation of the keybinds.

Repro steps:
1. Open Minecraft Preview and change controls. 
2. Open a editor project and move around.

Observed outcome:
Controls in the editor did not change.

Desired outcome:
To have my controls adapted to the controls set in the base game of Minecraft.

![keybinds](https://user-images.githubusercontent.com/16018643/223903029-64da0167-10a5-42d2-b28a-3818acac5b4a.png)
![WASD](https://user-images.githubusercontent.com/16018643/223913199-37e0cf83-5736-4de9-b764-ffb8a4669857.png)

**Comment by minecraftchloe** [comment]
Thank you for this write-up! I've created an item for us to investigate inheriting the vanilla keybindings for what we have mapped for WASD shift/space. And then we can also update the directions to be more inclusive of non-QWERTY keyboards! 

As far as individual client-side customization of keybindings for the Editor functional navigation, that is more complex than it seems ... but it is on our (long) wishlist 😅

**Comment by minecraftchloe** [comment]
We fixed this! Let us know what you think!
